src/cliffordsynthesis/Tableau.cpp

```diff
@@ -492,4 +492,107 @@ bool Tableau::isIdentityTableau() const {
   }
   return true;
 }
+
+Tableau Tableau::applyMapping(const std::vector<std::vector<bool>>* p) {
```

Comment:
No features of a pointer are used in this function. In this case it is better to just use a `const` reference.